chatgpt_wrapper/chatgpt.py

```diff
@@ -169,7 +172,10 @@ def get_history(self, limit=20, offset=0):
         }
         response = self._api_get_request(url, query_params)
         if response.ok:
-            return response.json()
+            history = {}
+            for item in response.json()["items"]:
+                history[item["id"]] = item
+            return history
         else:
             raise Exception(f"Failed to get history: {response.status} {response.status_text} {response.headers}")
```

Comment:
I suggest making this into a warning (see my comment here #133). I think that in general, the program should only dies if a non fixable error happens, otherwise it's best to inform the user and keep running.

Reply:
We can also catch these exceptions somewhere and make them into warnings